### ISSUE TITLE
printed structure does not obey --use-reflections

### ISSUE THREAD

**Issue by Jellby** [charnley/rmsd#78]
Using the `--use-reflections` option with the `calculate_rmsd` script affects the computed rmsd, but it doesn't seem to affect the output structure when `-p` is used too:

```
$ cat a.xyz
8

C     -0.75194898     0.15712980    -0.62038769
H     -1.32975650    -0.44037197    -1.31263409
H     -0.93962923     1.21050962    -0.77368922
C      0.75194846    -0.15712928    -0.62038611
H      0.93962586    -1.21050994    -0.77368854
H      1.32975920     0.44036972    -1.31263191
O     -1.10924506    -0.23012201     0.71758377
O      1.10924528     0.23012185     0.71758355

$ cat b.xyz 
8

C     -0.78959873     0.10620675    -0.52734402
H     -1.27819677    -0.85803167    -0.54888891
H     -0.99000386     0.63677309    -1.45273479
C      0.78962678    -0.10600531    -0.52735255
H      0.98992895    -0.63685564    -1.45258775
H      1.27843070     0.85811519    -0.54907319
O     -1.33661735     0.82461649     0.53999508
O      1.33642931    -0.82482111     0.53973588

$ calculate_rmsd a.xyz b.xyz 
0.8308965795175342

$ calculate_rmsd a.xyz b.xyz --use-reflections
0.3006966184789406

$ calculate_rmsd a.xyz b.xyz -p > b1.xyz

$ calculate_rmsd a.xyz b.xyz --use-reflections -p > b2.xyz

$ calculate_rmsd a.xyz b1.xyz
0.8308965795175342

$ calculate_rmsd a.xyz b2.xyz
0.8308965795175342

$ diff -s b1.xyz b2.xyz
Files b1.xyz and b2.xyz are identical

$ calculate_rmsd --version
rmsd 1.3.2

See https://github.com/charnley/rmsd for citation information
```

Moreover, with the latest pip version, `--use-reflections` does almost nothing at all:

```
$ calculate_rmsd --version
rmsd 1.4

See https://github.com/charnley/rmsd for citation information

$ calculate_rmsd a.xyz b.xyz 
0.830896579517534

$ calculate_rmsd a.xyz b.xyz --use-reflections
0.8308965795175339
```

**Comment by charnley** [comment]
Hi @Jellby, I finally found some time to look at this. The bug is 1.3.2 specific. It seems by default the reorder method was enabled by default. So not only did you check reflections, you checked reflections while reordering the atoms. To get the 0.3 RMSD in 1.4.0 use

    calculate_rmsd a.xyz b.xyz --use-reflections --reorder

The default `--reorder-method` is the same for both version, e.i. Hungarian-Distance.

However, can confirm **that reflections are not contained in the `-p` flag, as they should be**. I have started the fix https://github.com/charnley/rmsd/pull/80. Will need to add better tests for --print output, and merge ASAP.
